repos/system_upgrade/el7toel8/actors/persistentnetnamesconfig/libraries/persistentnetnamesconfig.py

```diff
@@ -2,6 +2,7 @@
 import os
 import re
 
+import leapp.libraries.common.config as leapp_config
```

Comment:
just one point, if I remember well, we told we do not want people to use "renaming" of libs for imports (unless you need it to resolve duplicate names).

Reply:
Should be fixed